033-OSSDatabaseMigration/Coach/05-online-migration.md

```diff
@@ -0,0 +1,223 @@
+# Challenge 5: Online migration
+
+[< Previous Challenge](./04-offline-cutover-validation.md) - **[Home](./README.md)** - [Next Challenge >](./06-online-cutover-validation.md)
+
+## Coach Tips
+
+The intent of this WTH is not to focus on networking so if the attendee has trouble configuring VPN/VNET peering, feel free to hand hold them through it
```

Comment:
Need a full stop at the end of this sentence.

Reply:
Fixed